src/task1.js

```diff
@@ -0,0 +1,5 @@
+(() => {
+    process.openStdin().addListener('data', data => { 
+        process.stdout.write('Output' + data.toString().split('').reverse().join('')+ '\n\n');
```

Comment:
Consider storing this output string in a separate constant to improve readability of your code.